LEDs.h

```diff
@@ -0,0 +1,14 @@
+#include <Arduino.h>
+#include <LedControl.h>
+
+#ifndef LEDs_H_
+#define LEDs_H_
+
+
+void LEDs__wake();
+void LEDs__start();
+
+//create LEDs Update function that takes CAN data and changes LEDs
+
+
+#endif LEDs_H_

```

Comment:
#ifndef, #define, and #endif directives should always have following strings capitalized.

LEDs_H_ should be LEDS_H_